schema/metrics/metrics.schema

```diff
@@ -0,0 +1,271 @@
+      "type": "array",
+      "items": {
+        "$ref": "/schemas/Bucket"
+      }
+    },
+    "positiveOffset": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      }
+    },
+    "negativeOffset": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      }
+    },
+    "zeroCount": {
+      "type": "number"
+    },
+    "scale": {
+      "type": "integer"
+    },
+    "max": {
+      "type": "number"
+    },
+    "min": {
+      "type": "number"
+    },
+    "sum": {
+      "type": "number"
+    },
+    "count": {
+      "type": "number"
+    },
+    "exemplars": {
+      "type": "array",
+      "items": {
+        "$ref": "/schemas/Exemplar"
+      }
+    },
+    "instrumentationScope": {
+      "$ref": "/schemas/InstrumentationScope"
+    },
+    "schemaUrl": {
+      "type": "string"
+    }
+  },
+  "required": [
+    "name",
+    "description",
+    "unit",
+    "kind",
+    "@timestamp"
+  ],
+  "$defs": {
+    "Bucket": {
+      "$id": "/schemas/Bucket",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "count": {
+          "type": "number"
+        },
+        "sum": {
+          "type": "number"
+        },
+        "max": {
+          "type": "number"
+        },
+        "min": {
+          "type": "number"
+        }
+      },
+      "required": [
+        "count",
+        "max",
+        "min"
+      ],
+      "title": "Bucket"
+    },
+    "Quantile": {
+      "$id": "/schemas/Quantile",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "quantile": {
+          "type": "number"
+        },
+        "value": {
+          "type": "number"
+        }
+      },
+      "required": [
+        "quantile",
+        "value"
+      ],
+      "title": "Quantile"
+    },
+    "InstrumentationScope": {
+      "$id": "/schemas/InstrumentationScope",
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        },
+        "schemaUrl": {
+          "type": "string"
+        },
+        "droppedAttributesCount": {
+          "type": "integer"
+        }
+      },
+      "title": "InstrumentationScope"
+    },
+    "Exemplar": {
+      "$id": "/schemas/Exemplar",
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "time": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "spanId": {
+          "type": "string"
+        },
+        "traceId": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "time"
+      ],
+      "title": "Exemplar"
+    },
+    "Attributes": {
+      "$id": "/schemas/Attributes",
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "data_stream": {
+          "$ref": "/schemas/Dataflow"
+        }
```

Comment:
Readme says attributes should just contain strings as values, one of the two should be changed

Reply:
will update